packages/manager/modules/iam/src/components/identities/groupIdentities/groupIdentities.controller.js

```diff
@@ -1,12 +1,15 @@
 export default class groupIdentitiesController {
   /* @ngInject */
-  constructor(IAMService) {
+  constructor(IAMService, $timeout) {
     this.IAMService = IAMService;
+    this.$timeout = $timeout;
   }
 
   $onChanges() {
     this.getIdentitiesGroups().then((data) => {
-      this.groups = data;
+      this.$timeout(() => {
+        this.groups = data;
+      });
```

Comment:
**nitpick:** To avoid `$timeout` here, can you replace `Promise.all` in https://github.com/ovh/manager/blob/9c94eccf235e9b3799b5f90381c07dedb1affee3/packages/manager/modules/iam/src/components/identities/groupIdentities/groupIdentities.controller.js#L17 with `$q.all`. It will fix the problem.